Automattic/jetpack
# My Jetpack: Add Extras interstitial page

It depends on https://github.com/Automattic/jetpack/pull/23035

This is a suggestion of adding an Extras interstitial page. Basically, we copied what the Jetpack plugin shows.
Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: Add Extras interstitial page

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to add extras interstitial page `https://your-testing-site/wp-admin/admin.php?page=my-jetpack#/add-extras`
* Check you see the Extras interstitial page :smart:
* Click on the `Add Jetpack Extras` will lead to the Jetpack dashboard

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/77539/155294275-cac83da5-e24d-4142-8cc1-779b28206e6d.png">
